docs/developers/schema_versions.md

```diff
@@ -24,3 +26,15 @@ To aid with this process, the following guidelines are provided:
 In general, a schema version should be adjusted if and only if there's a change in the schema's definition. This is typically reserved for major releases _(e.g. upgrading from iRODS 4.2 to iRODS 4.3)_.
 
 A _change in the schema definition_ is defined as the addition, modification, and/or removal of a JSON property or database table/column.
+
+### iRODS 5 and later
+
+!!! Important
+    Starting with iRODS 5, version numbers consist of three segments, _X.Y.Z_, where _X_ is the major version number, _Y_ is the minor version number, and _Z_ is the patch version number. A major version change consists of the major version number being incremented. For example, upgrading from iRODS 4.3.4 to iRODS 5.0.0 or iRODS 5.1.0 to iRODS 6.0.0.
```

Comment:
Could link to issue about versioning in iRODS.

That may be overkill.

Reply:
I feel a little skittish about linking GitHub issues in the docs, but that's just me. I'm sure I've done it before, though. :)

Reply:
I haven't formed an opinion about linking to issues yet, but I get what you're saying.

I think the message is clear enough and doesn't need a whole story.

Leaving as is.

Reply:
agreed - docs should be standalone with their own words i think.      there may be exceptions, but this seems a good default rule.